### DISCUSSION TITLE
Giesen WxA  (W6)  ET question

### DISCUSSION THREAD

**Discussion by redadder22** [artisan-roaster-scope/artisan#1252] [Q&A] [answered]
Hi,

For some reason the ET is displaying the gas % level instead of the ET value. Any idea why this is happening?

**Comment by bigl00z3** [comment]
Most likely that you are reading either the wrong device or port in:

- `Config/Devices...`   -->  Extra Devices tab
- `Config/Port...`  --> S7 tab

If you want others to be able to help, either post some screenshots of those windows (from the GitHub web page not by email) or share your artisan settings. You can export your settings via `Help/Save settings`. You will need to rename the `artisan-settings.aset` to a `.txt` extension instead of the `.aset` before uploading via the Github page.

Nico

**Reply by bigl00z3** [reply]
or you may have also change the target of the ET value in `Config/Devices...` --> "Symb ET/BT" tab and you are calling for the wrong ET Yn symbolic number (check the HELP menu of the "Symb ET/BT" tab for explanations)

**Reply by redadder22** [reply]
Many thanks. 
![Uploading Screenshot 2023-08-18 at 10.12.14.png…]()
<img width="958" alt="Screenshot 2023-08-18 at 10 12 31" src="https://github.com/artisan-roaster-scope/artisan/assets/29542458/b27f9329-3334-46e8-9282-ab55a1f524bc">

[artisan-settings.txt](https://github.com/artisan-roaster-scope/artisan/files/12377766/artisan-settings.txt)

**Reply by redadder22** [reply]
<img width="956" alt="Screenshot 2023-08-18 at 10 12 14" src="https://github.com/artisan-roaster-scope/artisan/assets/29542458/703a5d7a-9d37-482a-b747-5d54037a7df8">

**Reply by MAKOMO** [reply, marked as answer]
That symbolic assignment Y3 for ET instructs Artisan to take the values for this channel from the first channel of the first extra device. Just remove that Y3 assignment here.
